builder/assets/clever/_package.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>clever</name>
+  <version>0.0.1</version>
+  <description>Clever pseudopackage made for backwards compatibility</description>
+  <maintainer email="okalachev@gmail.com">Oleg Kalachev</maintainer>
+  <license>MIT</license>
+  <buildtool_depend>catkin</buildtool_depend>
```

Comment:
Shouldn't `catkin_make` yell at you for not adding all dependencies? Somehow it doesn't, but that's likely a catkin bug.
At the very least you should add message generation dependencies.

Reply:
Somehow the main package lacked `message_generation` dependency as well. Added both.